### DISCUSSION TITLE
Map doesn't show a lake as it should be

### DISCUSSION THREAD

**Discussion by WolfgangZ42** [devemux86/cruiser#38] [Q&A] [answered]
Really not important - but close to where I live, so it always catches my eye. 
The "Speichersee" northeast of Munich is not shown correctly in the loaded map - only with the online map:
![2023-02-17 16 11 25](https://user-images.githubusercontent.com/121806238/219695518-d3fb03a5-376c-40e3-b75c-77103caa82d4.png)   ![2023-02-17 16 12 34](https://user-images.githubusercontent.com/121806238/219695559-260f19a8-6bda-41d5-b11f-6bc120abde8c.png)
Mapsforge - style-motorideruk.zip/default . . . . . . Mapsforge - Biker
![2023-02-17 16 18 07](https://user-images.githubusercontent.com/121806238/219695611-92035b40-7594-42a9-92ff-514dbff6cdcf.png)
Online Map - OpenStreetMap

Can I change anything here?

**Comment by devemux86** [comment]
It seems to be related to OpenGL rendering in the VTM map library.

We have an open issue, I added your report: https://github.com/mapsforge/vtm/issues/1010
and see what can be improved.

**Comment by devemux86** [comment, marked as answer]
@WolfgangZ42 
If you would like to edit the theme file, you can solve this for your maps.
(we avoid it globally because it can cause problems on heavy water maps like Netherlands)

For example, edit the [Biker theme, line 295](https://github.com/mapsforge/vtm/blob/master/vtm-themes/resources/assets/vtm/biker.xml#L295) and add `mesh="true"` like:
```xml
<style-area fade="-1" fill="#99badf" id="water" mesh="true" />
```

![VTM](https://user-images.githubusercontent.com/3484020/219714566-817debcd-9c9b-42dd-bd22-05238c8af023.png)

**Reply by WolfgangZ42** [reply]
Emux, sure I'd like to change the "Biker-Theme". I did it for Kurviger.zip already - but where do I find Biker.zip or some other .zip with the xml-source? Thanks for a hint!

**Reply by devemux86** [reply]
@WolfgangZ42 

The latest Biker theme is always in VTM repository:
https://github.com/mapsforge/vtm/blob/master/vtm-themes/resources/assets/vtm/biker.xml

You just need the xml in the zip as the resources are included in Cruiser.
Unless you want to use your own icons.

You can edit and zip it, then open it with Cruiser.
(enable "Settings | Application | Advanced | Menu" or search settings for "menu")

**Reply by WolfgangZ42** [reply]
Thanks! Now it works. 😃

**Comment by MotoUKRider** [comment]
Nice to see you using my theme version!  All errors of mine are faithfully reproduced from the original source :)

I've applied that style tweak to mine, so here's the latest ZIP:
[style-motorideruk.zip](https://github.com/devemux86/cruiser/files/10771412/style-motorideruk.zip)

**Reply by WolfgangZ42** [reply]
> Nice to see you using my theme version!

At moment my preferred theme.

**Reply by WolfgangZ42** [reply]
Thanks a lot for the new version - just fine!
